include/crm/common/tags.h

```diff
@@ -22,11 +22,15 @@ extern "C" {
  * \ingroup core
  */
 
-//! Configuration tag object
+// Configuration tag object
+// @COMPAT Make internal when we can break API backward compatibility
+//!@{
+//! \deprecated Do not use (public access will be removed in a future release)
 typedef struct pe_tag_s {
     char *id;       //!< XML ID of tag
```

Comment:
Optionally remove the per-member exclamation points in this and later commits. You've been doing that lately for these struct/enum deprecations, so that the per-member comments won't show up in the generated public docs.

Reply:
Whoops, meant to -- fixed